front/lib/data_sources.ts

```diff
@@ -17,6 +17,9 @@ export function getProviderLogoPathForDataSource(
     case "slack":
       return `/static/slack_32x32.png`;
 
+    case "github":
```

Comment:
@lasryaric I tend to use these default `never` cases like below on line 23 to make sure we treat all values in an enum in a switch statement. Here it saved me (I had forgotten about this function). Do you know if there is a better way to do this ?

Reply:
Using a helper function could be a little cleaner: https://chat.openai.com/share/0c93321b-b553-430f-a06f-a6c82f56e4ee